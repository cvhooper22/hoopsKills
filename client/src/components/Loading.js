import React from 'react';

function Loading () {
  return (
    <svg version="1.0" xmlns="http://www.w3.org/2000/svg" width="1280px" height="831px" viewBox="0 0 12800 8310" preserveAspectRatio="xMidYMid meet" className="loader">
      <g id="oval" fill="#002255" stroke="none">
        <path d="M5960 8274 c-30 -2 -129 -8 -220 -14 -1534 -99 -2997 -577 -4040 -1320 -932 -664 -1493 -1483 -1631 -2381 -30 -192 -32 -571 -5 -754 87 -592 326 -1105 755 -1620 110 -132 397 -416 541 -535 1120 -927 2701 -1487 4495 -1590 394 -23 1036 -12 1445 26 2036 185 3808 1001 4757 2189 398 498 629 1035 692 1607 26 231 8 585 -40 822 -237 1166 -1224 2202 -2734 2869 -839 370 -1788 599 -2835 683 -182 15 -1034 28 -1180 18z"/>
      </g>
      <g id="y" fill="#ffffff" stroke="none">
        <path d="M5845 7579 c-341 -20 -743 -69 -1075 -130 -159 -29 -381 -77 -388 -84 -2 -2 38 -209 88 -461 50 -253 93 -468 95 -479 4 -21 5 -21 141 7 114 23 512 83 672 101 l42 5 0 -629 0 -630 -1227 -1390 c-674 -765 -1233 -1393 -1240 -1396 -34 -13 -582 365 -652 450 -7 9 -17 17 -22 17 -8 0 -120 -138 -530 -655 -48 -60 -85 -113 -82 -117 2 -4 63 -55 135 -113 489 -392 1156 -736 1905 -984 614 -203 1278 -337 1870 -376 102 -7 102 -7 107 16 4 19 106 972 106 992 0 2 -44 8 -97 12 -265 19 -713 79 -713 95 0 8 1980 2314 1989 2317 3 1 401 -476 883 -1060 658 -795 874 -1063 864 -1069 -47 -26 -545 -140 -799 -183 -38 -6 -38 -7 -33 -43 11 -71 85 -668 96 -772 6 -58 13 -124 16 -146 l5 -41 42 8 c743 142 1269 308 1859 588 374 177 820 453 1130 698 l67 53 -137 184 c-75 101 -204 274 -285 385 l-148 201 -87 -75 c-100 -87 -260 -202 -412 -297 -135 -84 -251 -151 -255 -146 -2 1 -527 641 -1168 1421 l-1164 1419 -6 261 c-4 144 -7 428 -7 631 l0 369 68 -7 c121 -12 716 -109 808 -132 18 -5 22 0 28 33 4 21 40 162 81 313 85 314 145 556 145 581 0 14 -26 23 -129 48 -393 94 -793 155 -1281 197 -218 19 -1069 27 -1305 13z"/>
      </g>
    </svg>
  );
}

export default Loading;
