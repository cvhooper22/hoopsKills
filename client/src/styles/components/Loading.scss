.loader {
  width: 65px;
  height: 42px;

  g#oval {
    animation: pulseToWhite 3s ease infinite;
    fill: $blue-color1;
  }

  g#y {
    animation: pulseToBlue 3s ease infinite;
    fill: white;
  }
}

@keyframes pulseToBlue {
  0% { fill: white; }
  50% { fill: $blue-color1; }
  100% { fill: white; }
}

@keyframes pulseToWhite {
  0% { fill: $blue-color1; }
  50% { fill: white; }
  100% { fill: $blue-color1; }
}