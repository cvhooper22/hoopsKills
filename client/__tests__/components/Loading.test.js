import React from 'react';
import renderer from 'react-test-renderer';
import Loading from '../../src/components/Loading';

describe('Loading -', () => {

  it('Should render Loading', () => {
    const component = renderer.create(<Loading />);

    expect(component).toMatchSnapshot();
  });

});