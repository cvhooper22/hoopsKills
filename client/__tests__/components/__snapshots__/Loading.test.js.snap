// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading - Should render Loading 1`] = `
<div
  className="loading"
>
  <div
    className="loading-icon"
  >
    <i
      className="ids-icon__spinner fa-spin"
    />
  </div>
</div>
`;
